app/components/bali/avatar/component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Bali
+  module Avatar
+    class Component < ApplicationViewComponent
+      renders_one :picture, Picture::Component
+
+      def initialize(form:,
+                     attr_model_name:,
```

Comment:
I think it should be named **method** to follow the same convention as the file_field helper: https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field

```suggestion
                     method:,
```